client/src/SaveModal.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Modal from "react-modal";
+import Button from "./Button";
+import TextInput from "./TextInput";
+
+Modal.setAppElement("#root"); //To remove error/warning in console for accessibility reasons
+function SaveModal() {
+  const [modalIsOpen, setmodalIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div className="absolute bottom-3 right-7">
+        <Button
+          text="Save"
+          fontSize="1.2em"
+          onClick={() => setmodalIsOpen(true)}
+        />
+      </div>
+      <div>
+        <Modal
+          className="m-auto w-max mt-20 text-black"
+          isOpen={modalIsOpen}
+          onRequestClose={() => setmodalIsOpen(false)}
+          style={{
```

Comment:
Can you explain a little bit about how this is working? Is this targeting an element `overlay`? Just curious about this piece